lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -197,20 +197,17 @@ def get_nets(self, org_name=None, org_id=None):
         r = self.request(path, method='GET')
         return r
 
-    def get_net(self, org_name, net_name, data=None):
-        """Return network information about a particular network."""
-        # TODO: Allow method to download data on its own
-        # if not data:
-        #     org_id = self.get_org_id(org_name)
-        #     path = '/organizations/{org_id}/networks/{net_id}'.format(
-        #         org_id=org_id,
-        #         net_id=self.get_net_id(
-        #             org_name=org_name,
-        #             net_name=net_name,
-        #             data=data)
-        #     )
-        #     return json.loads(self.request('GET', path))
-        # else:
+    # def get_net(self, org_name, net_name, data=None):
+    #     path = self.construct_path('get_all', function='network', org_id=org_id)
+    #     r = self.request(path, method='GET')
+    #     return r
+
+    def get_net(self, org_name, net_name, org_id=None, data=None):
```

Comment:
Beware that the fourth positional argument has shifted, so if people used this interface using positional arguments it may break. (I don't think there's a risk, as we can safely assume you're the only user of the interface...)